teams_accounts.go

```diff
@@ -286,3 +316,23 @@ func (api *API) TeamsAccountDeviceUpdateConfiguration(ctx context.Context, accou
 
 	return teamsDeviceResponse.Result, nil
 }
+
+// TeamsAccountConnectivityUpdateConfiguration updates zero trust account connectivity settings.
+//
+// API reference: https://developers.cloudflare.com/api/operations/zero-trust-accounts-patch-connectivity-settings
```

Comment:
```suggestion
// API reference: https://developers.cloudflare.com/api/operations/zero-trust-accounts-patch-connectivity-settings
```

Reply:
Thanks for reviewing @jacobbednarz, I'm not seeing a diff in the suggested change.